client/multi-currency-analytics/index.js

```diff
@@ -0,0 +1,69 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './styles.scss';
+
+addFilter(
+	'woocommerce_admin_report_table',
+	'woocommerce-payments',
+	( tableData ) => {
+		// If we don't need to or are unable to add the column, just return the table data.
+		if (
+			! tableData.items ||
+			! tableData.items.data ||
+			! tableData.items.data.length
+		) {
+			return tableData;
+		}
+
+		const addCurrencyColumn = 'orders' === tableData.endpoint;
```

Comment:
TODO: I can simplify this logic to just return `tableData` if the endpoint isn't `orders` prior to merge (originally, there were a few more things happening here)